Homework_2.playground/Contents.swift

```diff
@@ -1,47 +1,163 @@
-class Stack {
-    private let linkedList = LinkedList()
+/// Container for elements organized with LIFO principle.
+///
+/// First of all, for working with stack we need the instance of the particular type of Stack.
+///
+///     let stack = Stack<Int>()
+///
+/// Then we can provide operations for working with elements of the stack:
+///
+///     //add an element on top of the stack
+///     stack.push(element: 4)
+///
+///     //return an element on top of the stack
+///     print(stack.top())
+///     //prints "4"
+///
+///     //remove and return an element on top of the stack
+///     print(stack.pop())
+///     //prints "4"
+///
+///     //If Stack is empty pop() and top() return nil
+///     if let topElement = stack.pop() {
+///         prints(topElement)
+///     } else {
+///         print("Stack is empty")
+///     //prints "Stack is empty" because at the previous step
+///     //we pop the last element in the stack.
```

Comment:
Why did you write twice what the method does? Can you explain, please :) 
(in class and about each method separately)

Reply:
Done. I did a little cleanup in docs, now I think the docs look good.